transports/azure-service-bus/configuration.md

```diff
@@ -15,8 +15,7 @@ snippet: azure-service-bus-for-dotnet-standard
 
 These settings control how the transport connects to the broker.
 
-* `UseWebSockets()`: Configures the transport to use AMQP over websockets.
-* `TimeToWaitBeforeTriggeringCircuitBreaker(TimeSpan)`: The time to wait before triggering the circuit breaker after a critical error occurred. Defaults to 2 minutes.
+partial: connectivity
```

Comment:
Honestly I'm not a big fan of how the configuration page is structured with bullets today. I would prefer a structure similar to SQS but that would have been too much work to pull off so I continued sticking to the previous approach but made sure newer versions highlight the property syntax instead.

Reply:
We can raise an issue in this repo that, then it would be tackled by documentation work.